### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by GragonGit**
First commit LETS GOOOOOO xD

### CODE REVIEW

**Comment by TanConan** [comment]
Lookin good!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
```
Comment (TanConan) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Nice naming

Comment (GragonGit) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Thanks

Comment (GragonGit) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Do you think people will notice that we only write this to test the webhook?

**Comment by TanConan** [comment]
oops

**Review by TanConan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (GragonGit) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Just another test

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+about: Create a report to help us improve
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (GragonGit) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Comment 1

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (GragonGit) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: Comment 2

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
+assignees: ''
+
```
Comment (GragonGit) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: Comment 3

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (GragonGit) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: test

**Review by TanConan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (GragonGit) [reply .github/ISSUE_TEMPLATE/bug_report.md:7]: Hopefully the final test

**Pull request merged**